mycpp/gc_obj.h

```diff
@@ -0,0 +1,104 @@
+// Obj::heap_tag_ values.  They're odd numbers to distinguish them from vtable
```

Comment:
Please add `#include` guards on all header files

Looks like at least gc_heap.h and gc_obj.h don't have them

There is actually a `cpp/run.sh` script which has a boilerplate generator

Reply:
Fixed